src/libraries/System.Threading/src/System/Threading/Barrier.cs

```diff
@@ -91,7 +91,7 @@ protected BarrierPostPhaseException(SerializationInfo info, StreamingContext con
     /// completed.
     /// </para>
     /// </remarks>
-    [DebuggerDisplay("Participant Count={ParticipantCount},Participants Remaining={ParticipantsRemaining}")]
+    [DebuggerDisplay("Participant Count = {ParticipantCount}, Participants Remaining = {ParticipantsRemaining}")]
```

Comment:
```suggestion
    [DebuggerDisplay("ParticipantCount = {ParticipantCount}, ParticipantsRemaining = {ParticipantsRemaining}")]
```
Make names in debugger display match property names.